modalities/dom/components/elements/magenta-visualizer.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import Xen from '../xen/xen.js';
+
+class MagentaVisualizer extends Xen.Base {
+  get template() {
+    return Xen.html`
+      <canvas id="canvas" width="400" height="300"></canvas>
+      <button id="start" on-click="onStart"> Start </button>
```

Comment:
remove spaces around "Start" and "Stop". These words will be centered in the button.